collectors/ps_constants/core.py

```diff
@@ -45,9 +45,11 @@ def sync_contract_priority(source_dict):
     sync contract priority data
     """
     ContractPriority.objects.all().delete()
-    for ps_module, ps_component_list in source_dict.items():
-        for ps_component in ps_component_list:
-            ContractPriority(ps_module=ps_module, ps_component=ps_component).save()
+    for ps_module, ps_update_streams in source_dict.items():
```

Comment:
The **ps_module** seems to be unused so it could be just **_**.

Reply:
Or maybe even better you could use **values** instead of **items**.

Reply:
Expanded the comment so as to maintain the level of clarity (IMO) re. the difference in the source yaml structure and what's being stored.

Reply:
Fixed.